task-1/index.js

```diff
@@ -0,0 +1,113 @@
+/*Первая задача*/
+const sayHello = function (nameOfPerson) {
+	const greeting = 'Hello, ' + nameOfPerson + '!';
+	const greetingMark = 'Hi,' + nameOfPerson + '!';
+	if (nameOfPerson !== 'Mark') {
+		return greeting;
+	} else {
+		return greetingMark;
+	}
+};
+
+console.log(sayHello('Mark'));
+console.log(sayHello('Sergio'));
+/*====================================================================================================================================*/
+
+
+/*Вторая задача*/
+const findingHypotenuse = function (n, m) {
+	const hypotenuse = (((n ** 2) + (m ** 2)) ** (1 / 2));
+	return hypotenuse;
+};
+
+console.log(findingHypotenuse(3, 4));
+/*====================================================================================================================================*/
+
+
+/*Третья задача*/
+const min = function (a, b) {
+	return Math.min(a, b);
+};
+
+console.log(min(-100, 200));
+/*====================================================================================================================================*/
+
+
+/*Четвертая задача*/
+const isEven = function (n) {
+	return n % 2 === 0;
+};
+
+console.log(isEven(7));
+/*====================================================================================================================================*/
+
+
+/*Пятая задача*/
+const deleteChars = function (text) {
+	const str = text.slice(1, text.length - 1);
+	return str;
+};
+
+console.log(deleteChars('OartyomMikulichO'));
+/*====================================================================================================================================*/
+
+
+/*Шестая задача*/
+const someFn = function (stringValue) {
+	const capitalFirstLetter = stringValue[0].toUpperCase();
+	const restLetters = stringValue.slice(1).toLowerCase();
+	const capitalLastLetter = stringValue.slice(1, -1).toLowerCase() + stringValue.slice(-1).toUpperCase();
```

Comment:
1. naming issue. your variable is saying that it stores letter, but here you store more than letter.
 Suggestion: restCharsWithCapitalEnding
2. You need to do this operation only if initial string length isEven, in your case you do it every single time. Put this line in a proper place